python/libraries/bimvee/importNumpy.py

```diff
@@ -59,7 +59,28 @@ def importNumpy(filePathOrName, **kwargs):
     outDict['info']['filePathOrName'] = filePathOrName
 
     # Importing events
-    events = np.load(filePathOrName)
+    flow = np.load(filePathOrName)
+    outDict['data']['opticFlow'] = {}
+    outDict['data']['opticFlow']['flowMap'] = {}
+    outDict['data']['opticFlow']['flowMap']['flowMaps'] = np.swapaxes(flow, 1, 3)
+    step = 0.03
+    ts = np.arange(0, step * len(flow), step)
+    outDict['data']['opticFlow']['flowMap']['ts'] = ts
+
+    gt_file = os.path.join(os.path.dirname(filePathOrName), 'outdoor_day1_flow.npy')
```

Comment:
Could you make the path name an argument of the function? 'outdoor_day1_flow...' is too specific for something in a library of general functions.